app/models/device_group.rb

```diff
@@ -0,0 +1,7 @@
+class DeviceGroup < ApplicationRecord
+  has_many :devices, dependent: :destroy
+  belongs_to :organization, class_name: 'Organization', foreign_key: 'organization_id', optional: true
```

Comment:
no need to set `class_name` for cases when you follow naming convention on association name
same for the foreign_key